hasher-matcher-actioner/webapp/src/pages/SubmitContent.jsx

```diff
@@ -215,18 +219,33 @@ export default function SubmitContent() {
                     variant="primary"
                   />
                 </Collapse>
+                <Collapse in={!submittedId}>
+                  <Button
+                    style={{maxHeight: 38}}
+                    className="ml-3"
+                    variant="primary"
+                    disabled={submitting}
+                    type="submit">
+                    Submit
+                  </Button>
+                </Collapse>
                 <Collapse in={submittedId}>
-                  <Col className="ml-4">
-                    <Row>
-                      Submitted! It will take a few minutes for the hash to be
-                      generated.
-                    </Row>
-                    <Row>
-                      <Link to={`/matches/${submittedId}`}>
-                        Once created, the hash and any matches found can be
-                        viewed here.
-                      </Link>
-                    </Row>
+                  <Col>
+                    <Card>
+                      <Card.Header>Your content is submitted!</Card.Header>
+                      <Card.Body>
+                        <Button variant="primary" onClick={handleSubmitAnother}>
+                          Submit Another
+                        </Button>{' '}
+                        <Button
+                          variant="secondary"
```

Comment:
nit: I think the 'Submit Another' button is better as secondary. It feels weird having this one be gray so I would switch the order.